crates/ruff_python_trivia/src/tokenizer.rs

```diff
@@ -76,6 +76,7 @@ pub fn lines_after(offset: TextSize, code: &str) -> u32 {
                 cursor.eat_char('\n');
                 newlines += 1;
             }
+            ';' => continue,
```

Comment:
The function comment is outdated. 

The `lines_after_ignoring_trivia` is now no longer a more specific version of `lines_after` because it doesn't implement the "skip over commas". I think we should keep the two in sync. This possibly also applies to `lines_after_ignoring_end_of_line_trivia` but I would need to dig deeper into the actual usage.